src/components/Icons/NavigationArrow.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const NavigationArrow = ({
```

Comment:
Can you please follow the small/large convention we have established?
E.g. https://github.com/ticketmaster/aurora/blob/master/src/components/Icons/Home.js

Reply:
Done.